packages/discovery-provider/ddl/migrations/0036_add_challenge_disbursement_cap.sql

```diff
@@ -0,0 +1,12 @@
+begin;
+
+alter table challenge_disbursements 
+add column created_at timestamp with time zone default now();
```

Comment:
Is `timestamp with time zone` the type we use for all created_at fields?

Reply:
yep